CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# How to contribute to this project ?
+
+1. First you must choose a issue on [the project github](https://github.com/mdl29). If you want to do something but no issue were create yet you must create one (try to tag it properly).
+2. Fork the project.
+3. Create a new branch whith the following name format on your fork : `<issueNumber>_<descriptiveNameOfTheIssue>`.
+
+> Example : if you wanted to solve this issue (image below) you could have named your branch `138_Kickstarter_Yanis`.
```

Comment:
- `if you wanted to` -> `if you want to`
- `you could have named your branch` -> `you should  name your branch`